main.go

```diff
@@ -503,6 +509,28 @@ func validateParams() {
 	acrPassword := os.Getenv(acrPasswordKey)
 	gcrURLEnv := os.Getenv("gcrurl")
 
+	// initialize namespace inclusions and exclusions
+	if len(*argIncludeNamespaces) > 0 && len(*argExcludeNamespaces) > 0 {
+		logrus.Errorf("Ignoring 'exclude-namespaces', as 'include-namespaces' is provided.")
+		excludeNamespaces = []string{}
+	}
+
+	if len(*argIncludeNamespaces) > 0 {
+		includeNamespaces = strings.Split(*argIncludeNamespaces, ",")
+	} else {
```

Comment:
Not a great way of processing args and initializing inside the method `validate_params`, but going with the way things are setup up in the current state.